modules/camera/camera_win.h

```diff
@@ -30,14 +30,56 @@
 
 #pragma once
 
+#include "buffer_decoder.h"
 #include "servers/camera/camera_feed.h"
 #include "servers/camera_server.h"
+#include <initguid.h>
```

Comment:
```suggestion

#include <initguid.h>
```
Separate different include types